AQA_Lab_Task/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Driver;
+
+var driverGenerator = new DriverGenerator();
+var drivers = driverGenerator.GenerateDrivers();
+var vehicleGenerator = new VehicleGenerator(drivers);
+var vehicles = vehicleGenerator.GenerateVehicles();
+
+foreach (var driver in drivers)
+{
+    Console.WriteLine($"Driver {drivers.IndexOf(driver) + 1}");
+    driver.Print();
+}
+
+
+Console.Write("\nChoose driver (1-3): ");
+var choice = Console.ReadLine();
```

Comment:
Make a check for incorrect input. If its incorrect - ask user to choose driver again